### ISSUE TITLE
Styles don't seem to be taken into account

### ISSUE THREAD

**Issue by juba** [juba/robservable#28]
When CSS styles are defined with a <code>html\`<style>...</style>\`</code> cell, they don't seem to be taken into account when the notebook is imported into Observable.

I'll do a reprex soon.

**Comment by juba** [comment]
For example, here :

https://observablehq.com/@danburzo/color-vision-deficiency-simulation

For the plot to render we have to include both the chart cell and the `<style>` cell.

```r
robservable(
  "https://observablehq.com/@danburzo/color-vision-deficiency-simulation",
  include = c(5, 6)
)
```

 This is normal behavior but as `<style>` cells are generally not named, could there be a possibility to automatically detect and import them ?

**Comment by juba** [comment]
I'm not sure this would be possible, as the only way would be to get access to the cell code to check if it begins with <code>html\`<style>\`</code>.

**Comment by thomasballinger** [comment]
From my perspective, doing this might be helpful in common cases but would be inconsistent with other Observable embed tools, so might confuse users familiar with choosing Observable cells in other contexts. I agree it's a good bet that any CSS on the page should be applied, but this isn't always the case; especially when CSS from one embed might conflict with other HTML.

**Comment by juba** [comment]
Yes, you're certainly right, but if there is a way to automatically include `<style>` cells, I still wonder if it wouldn't be better to turn it on by default (with a possibility to disable it), as, as you say, in a vast majority of notebooks these styles are directly related to the main visualisation. So making visualisations more often work "out of the box" would be more beneficial that avoiding possible CSS conflicts, which could be solved by disabling automatic import if needed.

But I still don't see how to automatically import these cells, yet.

Anyway, thanks for taking the time to give your feedback here, it is really valuable and appreciated !
